src/utils/validators.js

```
@@ -22,11 +22,53 @@ const maskEmail = (email) => {
+    static isBlacklisted(email) {
+        const domain = email.split('@')[1]?.toLowerCase();
+        return blacklist.includes(domain);
+    }
+    static hasValidExtension(email) {
+        const domain = email.split('@')[1];
+        const extension = domain?.split('.').pop();
+        return validExtensions.includes(extension);
+    }
+    // E-posta kontrol fonksiyonu
+    static checkEmail(email) {
+        if (!this.isValidEmail(email)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)